Userland/Games/FlappyBug/Game.h

```diff
@@ -18,6 +18,61 @@
 
 namespace FlappyBug {
 
+struct Bug {
+    const float x { 50 };
+    const float radius { 16 };
+    const float starting_y { 200 };
+    NonnullRefPtr<Gfx::Bitmap> falling_bitmap;
+    NonnullRefPtr<Gfx::Bitmap> flapping_bitmap;
+    float y {};
+    float velocity {};
+
+    Bug(NonnullRefPtr<Gfx::Bitmap>&& falling_bitmap_value, NonnullRefPtr<Gfx::Bitmap>&& flapping_bitmap_value)
```

Comment:
```suggestion
    Bug(NonnullRefPtr<Gfx::Bitmap> falling_bitmap_value, NonnullRefPtr<Gfx::Bitmap> flapping_bitmap_value)
```
The copy will be optimized away if you move the NNRFs to the constructor because you also have `move` in the initializers.